WordPress/gutenberg
# Add missing label to border radius range control.

Fixes https://github.com/WordPress/gutenberg/issues/42098

## What?

Adds a missing label to the range input in the Border radius control.

## Why?

All form controls need to be labelled.

## How?

- adds a `label` prop with meaningful value
- adds `hideLabelFromVision` prop to make the label visually hidden
- updates docs and story of the range control: the `hideLabelFromVision` was added in a previous PR but it was undocumented

## Testing Instructions
- Edit a post and select a block that uses a Border radius control e.g. the Columns block.
- Check the Border radius control in the inspector sidebar.
- Check that:
  - the range input has a visually hidden `<label>` element associated with a `for` attribute to its ID
  - the range input has an `aria-label` attribute with the same text of the `<label>` element

Optionally:
- launch the story book: `npm run storybook:dev`
- check the Range control component
- check that toggling the `hideLabelFromVision` prop toggles the label visibility

Note: using both a `<label>` element and an `aria-label` attribute is redundant. The current component just works this way and a refactoring is out of the scope of this PR.

## Screenshots or screencast
